package.json

```diff
@@ -38,16 +38,16 @@
   ],
   "author": "Holepunch Inc.",
   "license": "Apache-2.0",
-  "gypfile": true,
   "bugs": {
     "url": "https://github.com/holepunchto/udx-native/issues"
   },
   "homepage": "https://github.com/holepunchto/udx-native#readme",
   "dependencies": {
     "b4a": "^1.5.0",
+    "bare-compat-napi": "^1.2.2",
```

Comment:
@telamon This should be a `devDependency`.

Reply:
oh gosh, yes it should!